ocs_ci/framework/main.py

```diff
@@ -8,45 +9,36 @@
 from ocs_ci.utility import utils
 
 
-def get_param(param, arguments, default=None):
-    """
-    Get parameter from list of arguments. Arguments can be in following format:
-    ['--parameter', 'param_value'] or ['--parameter=param_value']
-
-    Args:
-        param (str): Name of parameter
-        arguments (list): List of arguments from CLI
-        default (any): any default value for parameter (default: None)
-
-    """
-    for index, arg in enumerate(arguments):
-        if param in arg:
-            if '=' in arg:
-                return arg.split('=')[1]
-            return arguments[index + 1]
-    return default
-
-
 def init_ocsci_conf(arguments=None):
     """
     Update the config object with any files passed via the CLI
 
     Args:
         arguments (list): Arguments for pytest execution
     """
-    if not arguments:
-        return
-    custom_config = get_param('--ocsci-conf', arguments)
-    cluster_config = get_param('--cluster-conf', arguments)
-    if custom_config:
-        with open(os.path.expanduser(custom_config)) as file_stream:
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument('--ocsci-conf', action='append', default=[])
+    # cluster-conf parameter will be deleted once we will update all the jobs
+    parser.add_argument('--cluster-conf')
+    args, unknown = parser.parse_known_args()
```

Comment:
This should fix the failing unit tests.
```suggestion
    args, unknown = parser.parse_known_args(args=arguments)
```